.changeset/nice-drinks-do.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/design-system-codemods": minor
+"@hashicorp/design-system-components": minor
+---
+
+Placeholder
```

Comment:
Don't forget to update the changeset – otherwise I think we're ready to merge this